bundles/org.dataflowanalysis.analysis.converter/src/org/dataflowanalysis/analysis/converter/DataFlowDiagramConverter.java

```diff
@@ -0,0 +1,432 @@
+                if (abstractAssignment instanceof ForwardingAssignment) {
+                    for (Pin inPin : abstractAssignment.getInputPins()) {
+                        builder.append("forward ").append(mapInputPinToFlowName.get(inPin)).append("\n");
+                    }
+                } else {
+                    Assignment assignment = (Assignment) abstractAssignment;
+                    String value = getTermValue(assignment.getTerm()) ? "TRUE" : "FALSE";
+
+                    for (Label label : assignment.getOutputLabels()) {
+                        try {
+                            builder.append("set ").append(((LabelType) label.eContainer()).getEntityName()).append(".").append(label.getEntityName())
+                                    .append(" = ").append(value).append("\n");
+                        } catch (IllegalArgumentException ex) {
+                            System.out.println(
+                                    "Caution!! WebEditor cant handle complex Assignments yet. Only TRUE or NOT(TRUE) supported. Everything else ignored");
+                        }
+                    }
+                }
+            }
+            return builder.toString().trim();
+        }
+        return null;
+
+    }
+
+    // Currently only supports True or False
+    private boolean getTermValue(Term term) throws IllegalArgumentException {
+        if (term instanceof TRUE)
+            return true;
+        if (term instanceof NOT) {
+            return !getTermValue(((NOT) term).getNegatedTerm());
+        }
+        throw new IllegalArgumentException();
+    }
+
+    private void parseBehavior(Node node, Pin outpin, String lines, DataFlowDiagram dfd, DataDictionary dd) {
+        String[] behaviorStrings = lines.split("\n");
+        var behavior = node.getBehaviour();
+        for (String behaviorString : behaviorStrings) {
+            if (behaviorString.contains("forward ")) {
+                String packet = behaviorString.split(" ")[1];
+                Pin inpin = null;
+                for (Flow flow : dfd.getFlows()) {
+                    if (flow.getDestinationNode() == node) {
+                        if (flow.getEntityName().equals(packet)) {
+                            inpin = flow.getDestinationPin();
+                        }
+                    }
+                }
+
+                var assignment = ddFactory.createForwardingAssignment();
+                assignment.setOutputPin(outpin);
+                assignment.getInputPins().add(inpin);
+                behavior.getAssignment().add(assignment);
+            } else if (behaviorString.contains("set ")) {
+                String regex = "\\bset\\s+(\\S+)\\s+=\\s+(.+)\\b";
+
+                Pattern pattern = Pattern.compile(regex);
+                Matcher matcher = pattern.matcher(behaviorString);
+                matcher.find();
+
+                String variable = matcher.group(1);
```

Comment:
Indexing into groups is unsafe, if the amount of groups is not checked before.
Please return an error, if the number of groups is incorrect